examples/conversation_tone_analyzer_integration/tone_conversation_integration.v1.py

```diff
@@ -29,11 +30,11 @@
 
 # This example stores tone for each user utterance in conversation context.
 # Change this to false, if you do not want to maintain history
-maintainToneHistoryInContext = True
+global_maintainToneHistoryInContext = True
 
 # Payload for the Watson Conversation Service
 # user input text required - replace "I am happy" with user input text.
-payload = {
+global_payload = {
     'workspace_id': workspace_id,
     'input': {
         'text': "I am happy"
```

Comment:
Line 47 under `invokeToneConversation` says:

`invokeToneConversation calls the the Tone Analyzer service to get the`

extra `the` is not intentional. right?